File: Easy/1-Two-Sum/Solutions.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        cek = {}
        for index, value in enumerate(nums):
            selisih = target - value
            if selisih in cek:
                return [cek[selisih], index]
            cek[value] = index
                

File: Easy/27-Remove-Element/Solutions.cpp
class Solution
{
public:
    int removeElement(vector<int> &nums, int val)
    {
        int k = 0;
        for (int one : nums)
        {
            if (one != val)
            {
                nums[k] = one;
                k++;
            }
        }
        return k;
    }
};